src/screens/hardware-wallets/PairHardwareWalletErrorSheet.tsx

```diff
@@ -1,21 +1,26 @@
 import * as i18n from '@/languages';
 import React from 'react';
 import { Box, Inset, Stack, Text } from '@/design-system';
-import { CancelButton } from '@/screens/hardware-wallets/components/CancelButton';
+// import { CancelButton } from '@/screens/hardware-wallets/components/CancelButton';
 import { Layout } from '@/screens/hardware-wallets/components/Layout';
+// eslint-disable-next-line no-restricted-imports
 import { RouteProp, useRoute } from '@react-navigation/core';
```

Comment:
Actually, this eslint command is right, you should import `useRoute` from `@react-navigation/native`
It can cause serious problems when imported from the `core` module